spinn_front_end_common/interface/provenance/db.sql

```diff
@@ -218,3 +218,31 @@ CREATE TABLE IF NOT EXISTS boards_provenance(
     ip_addres STRING NOT NULL,
     ethernet_x INTEGER NOT NULL,
     ethernet_y INTEGER NOT NULL);
+
+---------------------------------------------------------------------
+-- A table to store log.info
+CREATE TABLE IF NOT EXISTS p_log_provenance(
```

Comment:
Should this table include the timestamp as its own column?

Reply:
good question I dont know

Reply:
Done